### PR TITLE
Update buildings card

### PR DESCRIPTION
**Pull request by anyatokar**
**Trello:** https://trello.com/c/vNESK4Rk

**Before:** 
Screen Shot 2021-04-25 at 10 21 38 PM
**After:**
Screen Shot 2021-04-25 at 10 12 00 PM

**Changes summary:**
- building address links to new Google Maps tab
- phone num is recognized for easy dialing
- lists a second phone number if it exists
- improved general look of card using Bootstrap-React ListGroup and Header features of Card component

**Next steps:**
- mirror changes for Saved Homes card (which is similar but not the same)

### CODE REVIEW

File: src/components/AllBuildingsCard.tsx [hunk 1]
```diff
@@ -53,67 +54,83 @@ export function AllBuildingsCard(props: IBuilding) {
...
+                href={mapViewUrl}
+                target="_blank" 
+                rel="noreferrer">
+              {streetNum} {street}
+              <br></br>
+              {city}, {state} {zip}
+            </a>
             <br></br>
-            Total MFTE Units: {totalRestrictedUnits}
+            <a href="tel:{phone}">
```
Comment (mmcknett) [inline src/components/AllBuildingsCard.tsx:119]:
```suggestion
            <a href=`tel:${phone}`>
```

File: src/components/AllBuildingsCard.tsx [hunk 2]
```diff
@@ -53,67 +54,83 @@ export function AllBuildingsCard(props: IBuilding) {
...
+            </a>
+            { phone2 ? (
+              <>
+                <br></br>
+                <a href="tel:{phone2}">
+                  {phone2}
+                </a>
+              </> 
+              ) : ('') 
+            }
```
Comment (mmcknett) [inline src/components/AllBuildingsCard.tsx:130]:
```suggestion
            {
              phone2 &&
              <>
                <br></br>
                <a href=`tel:${phone2}`>
                  {phone2}
                </a>
              </>
            }
```

**Review by mmcknett — CHANGES_REQUESTED** [review]
It looks like the links are making the url `tel:{phone}` instead of the actual phone number.
image

**Comment by anyatokar** [comment]
Good catch, thank you!

**Pull request merged**